Homework4.py

```diff
@@ -0,0 +1,57 @@
+"""task 1"""
+
+
+def solution_a(A):
```

Comment:
easy solution 

    for i in range(1, len(l) + 2):
        if i not in l:
            return i